src/battle-scene.ts

```diff
@@ -1245,8 +1242,15 @@ export default class BattleScene extends SceneBase {
         onComplete: () => {
           this.clearPhaseQueue();
 
+          // destroying those containers will call 'container.removeAll(true)', destroying all their children as well
+          this.uiContainer.destroy(true);
+          this.field.destroy(true);
+          this.fieldUI.destroy(true);
```

Comment:
Got a [reply](https://phaser.discourse.group/t/what-does-the-fromscene-parameter-of-gameobject-destroy-do/15248/2?u=nightkev) from a Phaser dev, the `fromScene` parameter shouldn't be set to `true` when `.destroy()` is called outside of the internal Phaser functions.
```suggestion
          this.uiContainer.destroy();
          this.field.destroy();
          this.fieldUI.destroy();
```

Reply:
sweet, that makes things easier, thank you!